src/AlertStack.js

```diff
@@ -12,8 +13,10 @@ import { layers } from './theme.js'
  * @example import { AlertStack } from '@dhis2/ui-core'
  * @see Live demo: {@link /demo/?path=/story/alertstack--default|Storybook}
  */
-const AlertStack = ({ className, children }) =>
-    createPortal(
+const AlertStack = ({ className, children, zIndex }) => {
+    const computedZIndex = useLayerBasedZindex(zIndex, layers.alert)
```

Comment:
The `alert` layer should _never_ be incremented, it is the absolute highest layer in our design system. It is the literal top.

Reply:
I think the alert stack warrants a hardcoded `layers.alert`.

Reply:
Yeah... I was wondering about that one whilst changing it. I've reverted these changes in https://github.com/dhis2/ui-core/pull/582/commits/3554b13bfdbd2f2b9632692941fedec9f59067ae